kroxylicious/src/main/java/io/kroxylicious/proxy/internal/package-info.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/**
+ * <b>io.kroxylicious.proxy.internal</b> contains Kroxylicious internal code implementing Proxying functionality using Netty.
+ * <h2>Understanding the Kroxylicious Pipeline</h2>
+ * The Kroxylicious pipeline for a client connection is composed of:
+ * <ul>
+ *     <li>A {@link io.kroxylicious.proxy.internal.KafkaProxyInitializer} that initialises the incoming channel pipeline</li>
+ *     <li>A {@link io.kroxylicious.proxy.internal.KafkaProxyFrontendHandler} that establishes an outgoing channel pipeline to the upstream broker</li>
```

Comment:
maybe mention the responsibility to forward requests from the downstream to the upstream channel.  I am talking about the work of io.kroxylicious.proxy.internal.KafkaProxyFrontendHandler#forwardOutbound.

Reply:
I think adding these summary sentences to the javadoc of KafkaProxyInitializer, KafkaProxyFrontendHandler, and KafkaProxyBackendHandler would be an incremental improvement too.